aws/src/main/java/org/apache/iceberg/aws/AwsClientFactories.java

```diff
@@ -50,7 +51,16 @@ public static AwsClientFactory defaultFactory() {
   public static AwsClientFactory from(Map<String, String> properties) {
     String factoryImpl = PropertyUtil.propertyAsString(
```

Comment:
one point missing here, based on the value `glue.lakeformation-enabled`, we should add `String factoryImplDefault` to either LF factory or the default factory.